src/swift-versions.ts

```diff
@@ -3,6 +3,7 @@ import * as core from "@actions/core";
 import { System, OS } from "./os";
 
 const VERSIONS_LIST: [string, OS[]][] = [
+  ["5.7.1", [OS.MacOS, OS.Ubuntu]],
```

Comment:
sorry for bumping in, but shouldn't this be
```suggestion
  ["5.7.1", OS.all()],
```
?
Otherwise I think https://github.com/swift-actions/setup-swift/blob/34ff090f791818966ee5d516e63e297a423232d8/.github/workflows/ci.yml#L33 cannot really work, right?